src/database-helper.ts

```diff
@@ -77,6 +77,30 @@ export async function processRequest(snap: DataSnapshot, pool: AccountPool, conf
   }
 }
 
+export async function fundBigFaucet(pool: AccountPool, config: NetworkConfig) {
```

Comment:
Does it matter what this function returns? I ask because right now it's either a promise or nothing (if it hits the `catch` block).

Reply:
it does not matter